spec.md

```diff
@@ -1198,6 +1198,14 @@ Service Binding](#fetching-a-service-binding) endpoint. However, in order for
 the Platform to successfully use the Service Binding, the information MUST be
 returned at least once.
 
+In some cases, the creation of a Service Binding has to be accompanied by
+additional network configuration. This could be the configuration of a firewall
+or a load balancer or it could be the setup of a network tunnel or a VPN
+connection.
+To enable the Platform or another central component to do this in a service
```

Comment:
I think `another central component` is confusing here, as it introduces a term which is not mentioned in the spec before. I think it would read better as `To enable the Platform to do this in a service agnostic way`